packages/components/pie-button/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import { terser } from 'rollup-plugin-terser';
+
+// `npm run build` -> `production` is true
+// `npm run dev` -> `production` is false
```

Comment:
Should these say yarn or are these not relevant at all as they aren't in the NPM scripts?

```suggestion
// `yarn build` -> `production` is true
// `yarn dev` -> `production` is false
```